cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt

```diff
@@ -57,9 +57,10 @@ internal fun Map<String, Any?>.toCCPAUserConsent(uuid: String?, applies: Boolean
 
     val status: CcpaStatus = getFieldValue<String>("status")
         ?.let { s ->
-            CcpaStatus.values().find {
+            val status = CcpaStatus.values().find {
                 it.name == s
             }
+            return@let status ?: CcpaStatus.unknown
```

Comment:
you don’t need to add  the return statement, just add `?: CcpaStatus.unknown` at the ent of find and remove the status property